web-console/knikubevirt/guest-agent-not-installed/guest-agent-not-installed.md

```diff
@@ -0,0 +1,22 @@
+#Guest Agent not installed
+
+The purpose of this document is to show what a VM would look like without the guest agent installed. 
+
+In the list view we show an “info” icon with the status with a tooltip that show that the guest agent isn’t installed. Further information can be provided in a tooltip.
+
+For the IP address, when the GA ins't installed, items will get “unavailable” text and a tooltip with more info on hover.
+
+![vm list](img/guest-agent-1.png)
+ 
+![vm list](img/guest-agent-2.png)
+
+![vm list](img/guest-agent-3.png)
```

Comment:
Here's an example of an unavailable state elsewhere in the Console. The text should probably be the same light gray color and maybe be "Not available" to slightly hint that it _could_ be if the guest agent were installed?

<img width="543" alt="2019-05-14 10 34 34" src="https://user-images.githubusercontent.com/9122899/57709748-949c5e80-7639-11e9-8527-fdf5ddab3ba5.png">

Reply:
I wonder if this screen/behavior when hovering over an unavailable IP address is necessary, especially as we move away from using tooltips on hover in general. Would the Status cell's icon and blue text be enough?

Reply:
+1 I should have kept reading